.github/workflows/ockam-healthcheck.yml

```diff
@@ -0,0 +1,63 @@
+name: Ockam Healthcheck Container Release
+
+on:
+  pull_request:
+    paths:
+      - '.github/actions/ockam-healthcheck.yml'
+      - 'implementations/elixir/ockam/ockam_healthcheck/**'
+  push:
+    paths:
+      - '.github/actions/ockam-healthcheck.yml'
+      - 'implementations/elixir/ockam/ockam_healthcheck/**'
+    branches:
+      - develop
```

Comment:
Let's also build image when there's an update to the [Healthcheck Dockerfile](https://github.com/build-trust/ockam/tree/develop/tools/docker/healthcheck/)